test/ManagementRareCytology/cases/G11AGCAISCytologyUnder25.yml

```diff
@@ -0,0 +1,32 @@
+---
+name: G1.2 Cytology Interpreted as AGC or AIS Without Atypical Endometrial Cells in Patient Under 25
```

Comment:
I believe this should say G1.1, right?

Reply:
Updated in https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/pull/124/commits/46dd0bcccaa2893ab939d49cb86550d4cbe725bf.